packages/server/graphql/InMemoryRateLimiter.ts

```diff
@@ -13,18 +18,25 @@ interface LastCall {
   [userId: string]: number
 }
 
-class RateLimiter {
-  private _records = {} as Records
-  private _lastCall = {} as LastCall
+export interface InMemoryRateLimiterConfig {
+  scheduleGc?: boolean
+}
+
+export class InMemoryRateLimiter {
```

Comment:
I retained some of the changes I made in the original PR so I could just port over the unit test directly.